runtime/watch_pod_test.go

```diff
@@ -159,12 +163,13 @@ func TestWatchPod(t *testing.T) {
 	for _, tc := range cases {
 		t.Run(tc.Name, func(t *testing.T) {
 			var watchErr error
-			podClient, podWatch := tc.PodClient()
+			clientset, podWatch := tc.PodClient()
 			descriptions := []string{}
 			done := make(chan bool)
+			k8scli.Client = clientset
 
 			go func() {
-				watchErr = WatchPod("test", "test", podClient, func(d string) {
+				watchErr = defaultWatchRunningBundle("test", "test", func(d string) {
 					fmt.Printf("NSK: got d -> %v\n", d)
```

Comment:
You can clip this if you're feeling generous. I can tell its my cruft thanks to my tag :P